peakina/readers/geojson.py

```diff
@@ -0,0 +1,16 @@
+from functools import wraps
+from typing import Any, Optional
+
+import geopandas as gpd
+
+
+@wraps(gpd.read_file)
+def read_file(
```

Comment:
maybe rename this function so it's explicit it's for geographic data

Reply:
alright :+1: